cmd/podman/machine/destroy.go

```diff
@@ -0,0 +1,91 @@
+package machine
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v3/cmd/podman/common"
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	"github.com/containers/podman/v3/pkg/machine"
+	"github.com/containers/podman/v3/pkg/machine/qemu"
+	"github.com/spf13/cobra"
+)
+
+var (
+	destroyCmd = &cobra.Command{
+		Use:               "destroy [options] NAME",
+		Short:             "Destroy an existing machine",
+		Long:              "Destroy an existing machine ",
+		RunE:              destroy,
+		Args:              cobra.ExactArgs(1),
+		Example:           `podman machine destroy myvm`,
+		ValidArgsFunction: completion.AutocompleteNone,
+	}
+)
+
+var (
+	destoryOptions machine.DestroyOptions
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
```

Comment:
Tunnel mode again - why are we enabling `podman-remote machine`?

Reply:
because that is the primary use case

Reply:
What? We want to create a virtual machine on the remote system?

Reply:
Yep, so the remote system can remote into the newly created VM